utils/rewrite-rules.yml

```
@@ -11,75 +11,55 @@ globals:
         chown haproxy: ${globals.haproxy_backup_file}
-    user: root
-  - cmd [bl]: |-
-        python3 /usr/local/bin/set-rewrite-rules.py "${globals.haproxy_frontend_file}" "${settings.rewrite-rules.toBase64()}"
-    user: root
-  - if ("${response.responses.errOut}" != ""):
-      - getRewritesRules
-      - return:
-          type: error
-          message: "An error occured when setting rewrite rules."
-          currentRules: "${globals.rewrites}"
+        if (! rpm --quiet -q jq);
+        then
+          yum install jq -y
+        fi
+        VAULT_TOKEN="${settings.vault_token}"
+        VAULT_HOSTNAME="${settings.vault_hostname}"
+        VAULT_SECRET_PATH="${settings.vault_secret_path}"
+        rewrite_rules=$(curl -s -H "X-Vault-Token: $VAULT_TOKEN" $VAULT_HOSTNAME/v1/kv/data/$VAULT_SECRET_PATH | jq -r .data.data[])
+        echo "$(awk -v rewriterules="$rewrite_rules" '/START_REWRITES/{print;print rewriterules;next}1' ${globals.haproxy_frontend_file})" > "${globals.haproxy_frontend_file}"
+      user: root
 
-  # Always return true cause if the cmd return -1, jelastic script just stop...
-  - cmd [bl]: |-
+  reloadHAProxy:
+    - cmd [bl]: |-
         service haproxy reload || true
-    user: root
-  - if ("${response.responses.errOut}" != ""):
-      - set:
-          error-out: "${response.responses.out} -- ${response.responses.errOut}"
-      - log: "reload failed"
-      - cmd [bl]: |-
-          mv  "${globals.haproxy_backup_file}" "${globals.haproxy_frontend_file}"
-      - getRewritesRules
-      - return:
-          type: error
-          message: "${this.error-out}"
-          currentRules: "${globals.rewrites}"
+      user: root
+    - if ("${response.responses.errOut}" != ""):
+        - set:
+            error-out: "${response.responses.out} -- ${response.responses.errOut}"
+        - log: "reload failed"
+        - cmd [bl]: |-
+            mv "${globals.haproxy_backup_file}" "${globals.haproxy_frontend_file}"
```

We still need to return an error with the haproxy reload error output as it was done in the previous version